lib/Base.js

```diff
@@ -3,96 +3,86 @@ const chalk = require('chalk');
 const yosay = require('yosay');
 const R = require('ramda');
 const _ = require('lodash');
-const prompts = require('./prompts');
 const RE = require('./ramda-extras');
-
+const {
+  prompts,
+  promptsOrder,
+  translateRequiredProp,
+  translateRequiredProps,
+} = require('./prompts');
 
 module.exports = class Base extends Generator {
-
-  copy(file) {
-    this.fs.copyTpl(
-      this.templatePath(file),
-      this.destinationPath(file),
-      this.props
-    );
-  }
-
   prompting() {
     R.pipe(
-      yosay, this.log
+      yosay,
+      this.log
     )(`Welcome to the primo ${chalk.red('generator-react-modules')} generator!`);
 
-    const desiredPrompts = R.props(this.prompts, prompts);
+    const promptForRequiredProps = R.pipe(
+      translateRequiredProps,
+      RE.uniqSortByArray(promptsOrder),
+      R.props(R.__, prompts),
+      this.prompt.bind(this)
+    );
 
-    return this.prompt(desiredPrompts)
+    return promptForRequiredProps(this.requiredProps)
       .then((props) => {
         this.props = props;
 
+        const shouldTranslateRequiredProp = prop => (
```

Comment:
I did, and then put it back. Decided not to just build the class for the sake of building the class. When anything else wants to use it, it'll be easy enough to throw onto the class as a method. But yeah, I agree with your thought- it's a general purpose method.